### DISCUSSION TITLE
need some help with installing blackarch over arch

### DISCUSSION THREAD

**Discussion by maximilien1990** [BlackArch/blackarch#4153] [Support]
im trying to install the blackarch tools on arch but i get this
unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description
warning: nfcutils: unknown key 'makepkgopt' in package description

for about a few hundred packages. whats the deal with that?

**Comment by noraj** [comment]
At what stage? What command did you issued?

**Reply by noraj** [reply]
Seems related to https://github.com/BlackArch/blackarch/issues/4133
